x-pack/plugins/apm/public/components/app/Settings/anomaly_detection/index.tsx

```diff
@@ -80,7 +84,7 @@ export const AnomalyDetection = () => {
       ) : (
         <JobsList
           status={status}
-          anomalyDetectionJobsByEnv={data.jobs}
+          jobs={data.jobs}
```

Comment:
No reason to pick a long name when we can pick a short :) (the context makes it obvious what type of job this is)